rllib/models/torch/torch_action_dist.py

```diff
@@ -638,10 +638,6 @@ def logp(self, x):
     def entropy(self):
         return self.dist.entropy()
 
-    @override(ActionDistribution)
-    def kl(self, other):
-        return self.dist.kl_divergence(other.dist)
-
     @staticmethod
     @override(ActionDistribution)
     def required_model_output_shape(action_space, model_config):
```

Comment:
How would this work with Tuples: "action_space.shape" ? the method "shape" is not defined in gym's spaces Tuples.